src/WebJobs.Script.WebHost/Diagnostics/IDeferredLogDispatcher.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
+{
+    public interface IDeferredLogDispatcher
```

Comment:
The interface can be removed if moving to `ILoggerProvider` approach. The draining of buffered logs can access the type directly - no contract needed.

Reply:
Removed IDeferredLogDispatcher